### ISSUE TITLE
Support for Falcon models

### ISSUE THREAD

**Issue by user-10389** [aws-neuron/transformers-neuronx#18]
Hey, is there any plan to add support for Falcon models ? 
That would be really great !

**Comment by mrnikwaws** [comment]
Thanks for the feature request @Mehdi-Aiyad I have tagged it in our system

**Comment by AWSGH** [comment]
yes, we do plan to test support for Falcon soon. Please keep track of this repo to stay up to date.

**Comment by Mehdi-Aiyad** [comment]
Thanks ! Will do !

**Comment by hannanjgaws** [comment]
Thank you for reaching out about using Neuron for Falcon.

We recommend that you enable this model using [Neuronx-Distributed-Inference](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/nxd-inference/index.html) (NxDI). NxDI is a PyTorch based library intended to provide developers with the ability to enable model parallelism/sharding techniques themselves for Neuron. Please check out our [tutorials](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/nxd-inference/tutorials/index.html) to see models that are already supported in NxDI and our [Developer Guides](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/nxd-inference/developer_guides/index.html) for how to onboard new models onto NxDI.  You are welcome to submit a PR with your contribution when you implement the model.
